### PR TITLE
#166816059 Setup SCSS and bootstrap frame work

### PR DESCRIPTION
**Pull request by dharmykoya**
#### What does this PR do?
Setup SCSS and bootstrap framework
#### Description of Task to be completed?
- To install style-loader
- To install CSS-loader
- To install bootstrap
- To install sass-loader
- To setup CSS files
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166816059 ](https://www.pivotaltracker.com/story/show/166816059)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: src/styles/partials/custom.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Define a color variable
+$color-sassy: #cc4397;
+$color-blue: #0059ff;
+
```
Comment (halimahO) [inline src/styles/partials/custom.scss]: Good job Dami, I noticed that there is a file named variables.scss for declaring all variables and here too in custom.scss, you declared those same variables. Can we have the variable declarations only in the files where they are needed or is there another use for the variables file?

Comment (dharmykoya) [reply src/styles/partials/custom.scss]: The purpose for the variables file is to have general styles, and call the variables when needed, anytime we need the variables in our component we import the variable file into that component. As for these lines, you pointed out I didn't remove these lines while I was refactoring into other files.
thank you.

Comment (halimahO) [reply src/styles/partials/custom.scss]: Okay..cool

**Review by tunedev — APPROVED** [review]
Good job, LGTM

**Pull request merged**